enterprise/frontend/src/embedding-sdk/lib/usage-problem.ts

```diff
@@ -11,21 +14,38 @@ interface SdkProblemOptions {
   hasTokenFeature: boolean;
 }
 
-const PROBLEMS = {
+export const USAGE_PROBLEM_MESSAGES = {
   API_KEYS_WITHOUT_LICENSE: `The embedding SDK is using API keys. This is intended for evaluation purposes and works only on localhost. To use on other sites, a license is required and you need to implement SSO.`,
   API_KEYS_WITH_LICENSE: `The embedding SDK is using API keys. This is intended for evaluation purposes and works only on localhost. To use on other sites, implement SSO.`,
   SSO_WITHOUT_LICENSE: `Usage without a valid license for this feature is only allowed for evaluation purposes, using API keys and only on localhost. Attempting to use this in other ways is in breach of our usage policy.`,
   CONFLICTING_AUTH_METHODS: `You cannot use both an Auth Provider URI and API key authentication at the same time.`,
   JWT_PROVIDER_URI_DEPRECATED: `The jwtProviderUri config property has been deprecated. Replace it with authProviderUri.`,
-
-  // TODO: this message is pending on the "allowing CORS for /api/session/properties" PR to be merged
   NO_AUTH_METHOD_PROVIDED: `You must provide either an Auth Provider URI or an API key for authentication.`,
 
+  // TODO: this message is not implemented yet, as it requires /api/session/properties to support CORS first.
   EMBEDDING_SDK_NOT_ENABLED: `The embedding SDK is not enabled for this instance. Please enable it in settings to start using the SDK.`,
```

Comment:
`NO_AUTH_METHOD_PROVIDED` actually works anywhere as it just checks the user-provided config field, the above comment is in the wrong place. moved it down one line.